### ISSUE TITLE
Release of training code

### ISSUE THREAD

**Issue by Vincentqyw** [SJTU-ViSYS/FeatureBooster#1]
Good work! Is there any plan to make the training code publicly available in the future?  Your work seems valuable for my ML research. Thanks.

**Comment by Antu3heng** [comment]
Thanks a lot for your interest! But unfortunately,  we do not intend to release the training code. We think you could refer to the training code for [D2-Net](https://github.com/mihaidusmanu/d2-net) and the loss function code for [FastAP](https://github.com/KevinMusgrave/pytorch-metric-learning/blob/master/src/pytorch_metric_learning/losses/fast_ap_loss.py), and it should be easy to re-implement the code.

**Comment by SamuelJamesFisher** [comment]
What about a model for disk features?

**Comment by Antu3heng** [comment]
Please see Issue #5.
